src/preferences/node/preference-server-module.ts

```diff
@@ -45,7 +46,7 @@ export const preferenceServerModule = new ContainerModule(bind => {
 
     bind<ConnectionHandler>(ConnectionHandler).toDynamicValue(ctx => {
         let clients: IPreferenceClient[] = []
-        ctx.container.get<IPreferenceServer>(IPreferenceServer).setClient({
+        const prefServer = ctx.container.get<IPreferenceServer>(IPreferenceServer).setClient({
```

Comment:
Yea you're right, why exactly is that and not of type IPreferenceServer like defaultServer, userServer and workspaceServer? Is it because of the scope/order of the bindings?